services/reddit/subreddit-subscribers.service.js

```diff
@@ -29,8 +28,6 @@ export default class RedditSubredditSubscribers extends BaseJsonService {
     },
   }
 
-  static _cacheLength = 7200
```

Comment:
Related to the question about what rate limit we get with authentication, are we confident that we'd have enough rate limit to get away with completely ditching this?